lxd/api_project.go

```diff
@@ -316,19 +324,36 @@ func projectPatch(d *Daemon, r *http.Request) response.Response {
 		req.Config["features.images"] = project.Config["features.profiles"]
 	}
 
+	_, err = reqRaw.GetString("limits.containers")
+	if err != nil {
+		req.Config["limits.containers"] = project.Config["limits.containers"]
+	}
+
+	_, err = reqRaw.GetString("limits.virtual-machines")
+	if err != nil {
+		req.Config["limits.virtual-machines"] = project.Config["limits.virtual-machines"]
+	}
+
```

Comment:
None of those actually do anything, that `GetString` will never return non-nil as returning non-nil would mean it was fed an invalid query in the first place. Same applies to the existing ones except for `description`, so they should also be removed.